src/utils.js

```diff
@@ -0,0 +1,3 @@
+// @flow
+export const escapeRegexTokens = (str: string): string =>
+  str.replace(/([.\-[])/g, '\\$1')
```

Comment:
I've found this robust regex on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#Escaping - maybe use this one with a comment where is it taken from?

Reply:
Good idea. I'll change that.